### PR TITLE
Fixed description of Sales Rep block

### PR DESCRIPTION
**Pull request by mnocon**
The Sales Rep block displays Sales Rep's details and has nothing to do with products.

Target: master, 4.6

Preview: https://ez-systems-developer-documentation--302.com.readthedocs.build/projects/userguide/en/302/content_management/block_reference/#sales-representative

### CODE REVIEW

File: docs/content_management/block_reference.md [hunk 1]
```diff
@@ -292,7 +292,7 @@ On the **Properties** tab, set values in the following fields:
 
 - **Name** – Enter a name for the page block.
 
-On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
+On the **Design** tab, in the **View** field, select the layout to be used to present the Sales representative's details and submit your changes.
```
Comment (juskora) [inline docs/content_management/block_reference.md:295]:
```suggestion
On the **Design** tab, in the **View** field, select the layout to be used to present the Sales representative's details page and submit your changes.
```

**Review by juskora — APPROVED** [review]

File: docs/content_management/block_reference.md (hunk 1 above)
Comment (juskora) [reply docs/content_management/block_reference.md:295]: But block is displayed as a section, we can go with the word: view.

Comment (mnocon) [reply docs/content_management/block_reference.md:295]: Makes sense, thanks!

Comment (mnocon) [inline docs/content_management/block_reference.md:295]:
```suggestion
On the **Design** tab, in the **View** field, select the layout to be used to present the Sales representative's details view and submit your changes.
```

Comment (mnocon) [reply docs/content_management/block_reference.md]: Added in https://github.com/ibexa/documentation-user/pull/302/commits/1c6d90424132982d8606bcff5c3bb9e88768813f

**Review by julitafalcondusza — APPROVED** [review]

**Pull request merged**